src/ideal/ideal.jl

```diff
@@ -1468,6 +1469,37 @@ function hilbert_series(I::sideal{spoly{T}}, w::Vector{<:Integer}, Qt::PolyRing)
    return Qt(new_ptr)
 end
 
+@doc raw"""
+    hilbert_series_data(I::sideal{spoly{T}}) where T <: Nemo.FieldElem
+
+Return the coeffcients of polynomial $Q(t)$ where $\frac{Q(t)}{\prod_i (1-t)}$
+is the Hilbert-Poincare series of $I$ for weights 1.
+The generators of $I$ must be given as a Groebner basis.
+The coefficients are of type BigInt
+"""
+function hilbert_series_data(I::sideal{spoly{T}}) where T <: Nemo.FieldElem
+   Qt,(t,) = polynomial_ring(ZZ, ["t"])
+   h = hilbert_series(I,Qt)
+   v = [convert(BigInt,c) for c in coefficients(h)]
```

Comment:
but as long as the coefficients in `h` are ints, this is pointless, isn't it?

Reply:
No, the polynomial h is in the ring Qt, i.e. its coefficients are in ZZ